pkgs/tools/misc/steampipe-packages/steampipe-plugin-kubernetes/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  steampipe,
+}:
+
+buildGoModule rec {
```

Comment:
suggestion: buildGoModule supports finalAttrs pattern now, so you can avoid the expensive `rec` if you want to.